libs/io/__init__.py

```
@@ -1,3 +1,3 @@
+from .helpers import get_connection_options
```

Ruff (0.12.2)

`.helpers.get_connection_options` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)